src/rt.erl

```diff
@@ -1213,9 +1209,12 @@ product(Node) ->
        true -> unknown
     end.
 
-try_nodes_ready([Node1 | _Nodes], 0, _SleepMs) ->
-    lager:info("Nodes not ready after initial plan/commit, retrying"),
-    plan_and_commit(Node1);
```

Comment:
Seems weird that we were planning and committing inside of the check for ready nodes